en.md

```diff
@@ -196,6 +196,8 @@ The processing of your personal data for the use of our enquiry and contact func
 
 The headquarters of Zendesk are located in the US, and your data, gathered when using our enquiry and contact function, are sent there. Zendesk is certified under the “Privacy Shield” US-EU data protection agreement and undertakes to uphold the EU data protection provisions. https://www.privacyshield.gov/participant?id=a2zt0000000TO6hAAG
 
+Help scout is located in the US too. They are GDPR compliant and have a solid security and privacy practices in place that go beyond the requirements of this new regulation. https://www.helpscout.com/company/legal/gdpr/
```

Comment:
I would remove "solid" here. This is a legal document and I think we should better refrain from publicly commenting on the quality of any security/privacy practices of a 3rd party. We might become liable in some shape or form. I would also remove "new" since its better to make sure this copy is time less and we don't have to update it again when its not "new" anymore.